packages/cfd/src/Containers/dynamic-leverage/dynamic-leverage-modal-content.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { Text } from '@deriv/components';
-import { localize } from '@deriv/translations';
+import { Localize, localize } from '@deriv/translations';
 import { dynamic_leverages } from '../../Constants/dynamic-leverage-content/dynamic-leverage-content';
 import { DynamicLeverageMarketCard } from './dynamic-leverage-market-card';
 
 const DynamicLeverageModalContent = () => (
     <div className='dynamic-leverage-modal'>
         <Text as='h4' size='s'>
-            {localize(
-                'Enjoy dynamic leverage of up to 1:1500 when trading selected instruments in the forex, commodities, cryptocurrencies, and stock indices markets. Our dynamic leverage adjusts automatically to your trading position, based on asset type and trading volume.'
-            )}
+            <Localize i18n_default_text='Enjoy dynamic leverage of' />
+            <Text as='span' size='s' weight='bold'>
+                {localize(' up to 1:1500 ')}
+            </Text>
+            <Localize i18n_default_text='when trading selected instruments in the forex, commodities, cryptocurrencies, and stock indices markets. Our dynamic leverage adjusts automatically to your trading position, based on asset type and trading volume.' />
```

Comment:
help to update this to a single component e.g.
`<Localize i18n_default_text={text} components={...} /> ;`

Reply:
Updated.